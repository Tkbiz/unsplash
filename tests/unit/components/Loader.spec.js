import {mount, createLocalVue} from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import Loader from '@/components/app/Loader'

Vue.use(Vuetify)

const localVue = createLocalVue()

describe('Loader.vue', () => {
  let vuetify
  let wrapper

  beforeEach(() => {
    vuetify = new Vuetify()
    wrapper = mount(Loader, {
      localVue,
      vuetify
    })
  })

  afterEach(() => {
    wrapper.destroy()
  })

  it('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  it('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
