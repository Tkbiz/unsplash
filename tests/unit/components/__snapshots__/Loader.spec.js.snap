// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue should match snapshot 1`] = `
<div class="container">
  <div class="row py-5 justify-center">
    <div class="lds-dual-ring"></div>
  </div>
</div>
`;
